packages/client/hmi-client/src/workflow/tera-workflow.vue

```diff
@@ -399,21 +282,6 @@ function appendOutputPort(
 		isOptional: false,
 		status: WorkflowPortStatus.NOT_CONNECTED
 	});
-
-	// FIXME: This is a bit hacky, we should split this out into separate events, or the action
-	// should be built into the Operation directly. What we are doing is to update the internal state
-	// and this feels it is leaking too much low-level information
-	if (node.operationType === WorkflowOperationTypes.CALIBRATION_CIEMSS) {
-		const state = node.state as CalibrationOperationStateJulia;
-		if (state.chartConfigs.length === 0) {
-			// This only ends up showing the output of the first run, perhaps we should consider showing
-			// the output of the last run, or all runs?
-			state.chartConfigs.push({
-				selectedRun: port.value[0],
-				selectedVariable: []
-			});
-		}
-	}
```

Comment:
This should be replaced by update-state emitter, it also doesn't make a lot of sense here - the predicate is on ciemss but operates on julia-state ... so ... chop chop.